live-examples/js-examples/set-prototype-foreach.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">function logSetElements(value1, value2, set) {
+  console.log('s[' + value1 + '] = ' + value2);
```

Comment:
the `Map` version of this used a template literal and I got all excited. Should we use it for both? (these are technically ES 2015, right?)

Reply:
yes ... yes I think we should :)